tests/integration/issues/github_1546/good3/__init__.py

```diff
@@ -25,7 +25,7 @@ def __init__(self,
         self.target_size = target_size
         self.channel_axis = channel_axis
 
-    def craft(self, blob: 'np.ndarray', *args, **kwargs) -> List[Dict]:
+    def segment(self, blob: 'np.ndarray', *args, **kwargs) -> List[Dict]:
```

Comment:
i dont see the need of changing `github_1546` test case, but changing also does no harm

Reply:
it was failing the test, I think is because Segmenter does not have craft anymore as method